features/step_definitions/pseud_steps.rb

```diff
@@ -0,0 +1,21 @@
+When /^I change the pseud "([^\"]*)" to "([^\"]*)"/ do |old_pseud, new_pseud|
+  step %{I edit the pseud "#{old_pseud}"}
+  fill_in("Name", with: new_pseud)
+  click_button("Update")
+end
+  
+When /^I edit the pseud "([^\"]*)"/ do |pseud| 
+  p = Pseud.where(name: pseud, user_id: User.current_user.id).first
+  visit edit_user_pseud_path(User.current_user, pseud)
+end
+
+When /^I add the pseud "([^\"]*)"/ do |pseud| 
```

Comment:
Ambiguous regexp literal. Parenthesize the method arguments if it's surely a regexp literal, or add a whitespace to the right of the `/` if it should be a division.<br>Trailing whitespace detected.